my-app/src/app/features/course-form-page/course-form-page.component.ts

```diff
@@ -48,17 +49,20 @@ export class CourseFormPageComponent implements OnInit, OnDestroy {
 
   private initEditMode(id: string) {
     this.isEditMode = true;
-    this.course = this.courseDataService.getCourseById(id);
-    this.breadcrumbsService.setLastLabel(this.course.title);
+    this.courseDataService.getCourseById(id).subscribe(course => {
+      this.course = course;
+      this.breadcrumbsService.setLastLabel(this.course.name);
+      this.cd.markForCheck();
```

Comment:
[optional] you could get rid of subscription and `markForCheck` call using stream of course and transformed stream of course name + `async` pipe